data-prepper-plugins/mutate-event-processors/src/test/java/org/opensearch/dataprepper/plugins/processor/mutateevent/ConvertEntryTypeProcessorTests.java

```diff
@@ -111,6 +113,30 @@ void testStringToIntegerConvertEntryTypeProcessor() {
         assertThat(event.get(TEST_KEY, Integer.class), equalTo(testValue));
     }
 
+    @Test
+    void testArrayOfStringsToIntegerConvertEntryTypeProcessor() {
```

Comment:
These are two different test cases. Please split into two `@Test` methods. You can put them in a `@Nested` class if you want to share the common `expectedResults`.

Something like:

```
@Nested
class CollectionTest {
   Integer testValue1;
   Integer testValue2;
   Integer testValue3;
  List<Integer> expectedResult;
  @BeforeEach 
  void setUp() {
        Integer testValue1 = random.nextInt();
        Integer testValue2 = random.nextInt();
        Integer testValue3 = random.nextInt();
        List<Integer> expectedResult = new ArrayList<>();
        expectedResult.add(testValue1);
        expectedResult.add(testValue2);
        expectedResult.add(testValue3);
  }
}
```